ctapipe/io/tableloader.py

```diff
@@ -64,13 +64,19 @@ class TableLoader(Component):
 
     input_url = traits.Path(directory_ok=False, exists=True).tag(config=True)
 
-    load_dl1_images = traits.Bool(False).tag(config=True)
-    load_dl1_parameters = traits.Bool(True).tag(config=True)
-    load_dl2_geometry = traits.Bool(False).tag(config=True)
-    load_simulated = traits.Bool(False).tag(config=True)
-    load_true_images = traits.Bool(False).tag(config=True)
-    load_trigger = traits.Bool(True).tag(config=True)
-    load_instrument = traits.Bool(False).tag(config=True)
+    load_dl1_images = traits.Bool(False, help="load extracted images").tag(config=True)
+    load_dl1_parameters = traits.Bool(
+        True, help="load reconstructed image parameters").tag(config=True)
+    load_dl2_geometry = traits.Bool(
+        False, help="load reconstructed shower geometry information").tag(config=True)
+    load_simulated = traits.Bool(
+        False, help="load simulated shower information").tag(config=True)
+    load_true_images = traits.Bool(
+        False, help="load simulated shower images").tag(config=True)
```

Comment:
does this include also the simulated shower image parameters?

Reply:
`load_true_images` means loading the simulated images, `load_simulated` means  loading the simulated shower information

the fact that I needed to specify it means that perhaps we should rename `load_simulated` into `load_simulated_showers` :)

Reply:
but where are the simulated parameters?  It is still not clear which options loads them.  They are not the same as the images.

Reply:
Not yet implement I think

Reply:
we don't have them I think - that would be another traitlet `load_true_parameters`

Reply:
ah no sorry we have them, but we should make it a different traitlet indeed

Reply:
Done.